pytorch_lightning/core/mixins/hparams_mixin.py

```diff
@@ -47,6 +47,10 @@ class ``__init__`` to be ignored
             frame: a frame object. Default is None
             logger: Whether to send the hyperparameters to the logger. Default: True
 
+        Raises::
+            ValueError:
```

Comment:
I guess this should be a `RuntimeError` or a `UserError` instead since it's not the value that's wrong.

Reply:
I'd suggest [here](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/models/test_hparams.py) :)